Recipes/Recipes.API/Models/Entities/IngredientEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Recipes.API.Models.Entities;
+public class IngredientEntity
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int Id { get; set; }
+
+    public string Name { get; set; }
```

Comment:
This seems wrong. Why is this added here? We have previously decided on a different entity without RecipeIngredient relationship, which is already implemented.

Reply:
The commits seemed to be messed up. No changes should be done in `Recipes/`

Reply:
Fixed commit issues